pkg/util/util.go

```diff
@@ -0,0 +1,114 @@
+// Package util provides utility functions for the sops-compliance-checker.
+package util
```

Comment:
I would use a different package name. `util` implies that it contains some less important utility functions, but it actually contains a relatively important bit: the config schema. So maybe `config` could be a getter alternative here.

Reply:
If you go for `config` as a package name, you can also rename the functions `LoadConfig` and `ValidateConfig` to `Load` and `Validate`, because they will read as `config.Load` and `config.Validate` at the callsite in other packages.

Reply:
I thought about it but was not sure if it makes sense to have a package just for config stuff. Maybe it makes sense if we put it under `/internal` ?

Reply:
Okay some of the stuff there maybe we want it to be public for now, if needed we can restructure later. 
I moved it under `pkg/config`

Reply:
I would prefer to hold off with this split just now. In the end this is "just" the user facing configuration schema. We may introduce another package for rules with more specific types and interfaces to ease internal processing which use a different representation than the user facing config.

Reply:
I was thinking that because we plan to introduce more specific types and interfaces that it makes sense to do the split. At the end the Rule type will still make sense to be in the rule package together with the other types, no?